src/qt/platformstyle.cpp

```diff
@@ -8,10 +8,8 @@
 
 #include <QApplication>
 #include <QColor>
-#include <QIcon>
 #include <QImage>
 #include <QPalette>
-#include <QPixmap>
```

Comment:
This is in the header BUT I believe it's not necessary there. IMO the first exercise should be to reduce to a minimal the includes in a header file.